.github/workflows/sphinxbuild.yml

```diff
@@ -0,0 +1,35 @@
+name: "Pull Request Docs Check"
```

Comment:
Awesome!!

Reply:
lol I came here to say we should switch to gh actions once we're on python3 :rocket: